backend/src/test/java/de/neuefische/koheis/backend/charactercards/CharacterCardIntegrationTest.java

```diff
@@ -0,0 +1,39 @@
+package de.neuefische.koheis.backend.charactercards;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class CharacterCardIntegrationTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @DirtiesContext
+    @Test
+    void expectEmptyListOnGet() throws Exception {
+        //GIVEN
+        //WHEN
+        mockMvc.perform(
+                        MockMvcRequestBuilders.get("/api/character_cards/cards")
+                )
+                //THEN
+                .andExpect(status().isOk())
+                .andExpect(content().json("""
+                            []
+                        """));
```

Comment:
you dont have to block string here writing:

```java
.andExpect(content().json("[]"));
```

is sufficient

Reply:
👍